beacon_node/execution_layer/src/lib.rs

```diff
@@ -30,6 +34,9 @@ mod engines;
 mod payload_status;
 pub mod test_utils;
 
+/// Name for the default file used for the jwt secret.
+pub const DEFAULT_JWT_FILE: &str = "jwtsecret";
```

Comment:
```suggestion
pub const DEFAULT_JWT_FILE: &str = "jwt.hex";
```

This would match the example in [the spec](https://github.com/ethereum/execution-apis/blob/main/src/engine/authentication.md#key-distribution), just a suggestion.